### ISSUE TITLE
pipe-cli: not-handled error while trying to execute commands with invalid config

### ISSUE THREAD

**Issue by NShaforostov** [epam/cloud-pipeline#750]
**Describe the bug**
If `pipe` config contains some invalid data (e.g. outdated invalid access token), then trying to execute any `pipe` command including `pipe --version` throws an error that is not handled:
```
~> pipe --version
Traceback (most recent call last):
  File "pipe-cli/pipe.py", line 1132, in <module>
  File "site-packages/click/core.py", line 722, in __call__
  File "site-packages/click/core.py", line 696, in main
  File "site-packages/click/core.py", line 621, in make_context
  File "site-packages/click/core.py", line 1018, in parse_args
  File "site-packages/click/core.py", line 880, in parse_args
  File "site-packages/click/core.py", line 1404, in handle_parse_result
  File "site-packages/click/core.py", line 78, in invoke_param_callback
  File "pipe-cli/pipe.py", line 54, in print_version
  File "pipe-cli/pipe.py", line 44, in silent_print_api_version
  File "pipe-cli/src/api/app_info.py", line 25, in info
  File "pipe-cli/src/api/base.py", line 45, in call
  File "site-packages/requests/api.py", line 75, in get
  File "site-packages/requests/api.py", line 60, in request
  File "site-packages/requests/sessions.py", line 524, in request
  File "site-packages/requests/sessions.py", line 637, in send
  File "site-packages/requests/adapters.py", line 516, in send
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='asdfasdf.sdf', port=443): Max retries exceeded with url: /sdfsdf/app/info (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7fc27be31e90>: Failed to establish a new connection: [Errno -2] Name or service not known',))
[338] Failed to execute script pipe
```

**Expected behavior**
More graceful and informative error message.

**Environment:**
 - Cloud Provider: AWS, Azure, GCP
 - Version 0.16.0.3059.b9dbe692746b52c70aba83e0b46c3a9d543638fe

**Comment by mzueva** [comment]
@NShaforostov Fixed via #772

**Comment by NShaforostov** [comment]
Verified
